src-frontend/app/controllers/procedure.js

```diff
@@ -19,9 +19,20 @@ export default Ember.Controller.extend({
         return pageModels;
     },
 
+    generateErrorString: function(errors, type) {
+        var message = 'Something went wrong creating the ' + type;
+        errors.forEach(function(error){
+            for (var attr in error.details) {
+                message = attr.toString() + ": " + error.details[attr][0];
```

Comment:
It should be double quotes for UI strings. We only use single quotes for key-like strings